src/Catalyst.Core.Modules.Web3/Controllers/EthController.cs

```diff
@@ -50,15 +51,21 @@ public EthController(IWeb3EthApi web3EthApi, IWeb3HandlerResolver handlerResolve
         [HttpPost]
         public JsonRpcResponse Request([FromBody] JsonRpcRequest request)
         {
-            _logger.Information("ETH JSON RPC request {id} {method} {params}", request.Id, request.Method, request.Params);
             EthWeb3RequestHandlerBase handler = _handlerResolver.Resolve(request.Method, request.Params.Length);
             if (handler == null)
             {
-                return new JsonRpcErrorResponse {Result = null, Error = new Error {Code = (int) ErrorType.MethodNotFound, Data = $"{request.Method}", Message = "Method not found"}};
+                JsonRpcErrorResponse errorResponse = new JsonRpcErrorResponse {Result = null, Error = new Error {Code = (int) ErrorType.MethodNotFound, Data = $"{request.Method}", Message = "Method not found"}};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Split this 211 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4979568)